scripts/run.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import print_function
+from subprocess import call
+import os
+
+
+def run():
+    if not os.path.exists('../conf/osdev.bochsrc'):
+        raise Exception('Missing bochs config!')
+
+    if not os.path.exists('../bin/floppy.bin'):
+        raise Exception("Missing disk image.\nYou should run 'python manage.py build' first.")
+
+    print("Running bochs...")
+    if call('bochs -f "../conf/osdev.bochsrc"', shell=True) is not 0:
```

Comment:
Comparing with `is` is error-prone (compares object identity — works only by chance or because of a CPython implementation detail), use `!=`.

But in this case, you can just use [`subprocess.check_call()`](https://docs.python.org/2.7/library/subprocess.html#subprocess.check_call) which raises an exception for you.